src/include/sof/dai.h

```diff
@@ -64,6 +64,7 @@ struct dai_ops {
 	int (*pm_context_store)(struct dai *dai);
 	int (*probe)(struct dai *dai);
 	int (*set_loopback_mode)(struct dai *dai, uint32_t lbm);
+	int (*get_loopback_mode)(struct dai *dai);
```

Comment:
Please remove set_loopback_mode and use an int (*ioctl)(*dai, int cmd, void *data)  callback for this.

Reply:
OK, will refine DAI.